### DISCUSSION TITLE
Collaborative Editing

### DISCUSSION THREAD

**Discussion by mikozet** [requarks/wiki#5506] [Help / Questions] [answered]
Hi,

Is there a way to turn on/add/activate collaborative editing for .md files? 
Our team plans to switch to wiki.js and it is something which we really missing during our tests.

Best,

**Comment by NangeGao** [comment]
Up! The same to us.

**Comment by NGPixel** [comment, marked as answer]
Follow https://requarks.canny.io/wiki/p/real-time-editingcollaboration-like-gdocs
